library/alloc/src/boxed.rs

```diff
@@ -1211,8 +1211,16 @@ impl<T: ?Sized, A: Allocator> Box<T, A> {
 
 #[stable(feature = "rust1", since = "1.0.0")]
 unsafe impl<#[may_dangle] T: ?Sized, A: Allocator> Drop for Box<T, A> {
+    #[inline]
```

Comment:
Idk, maybe that inline increased binary size?

Reply:
`box_free` was also marked as inline, so I don't think that is the cause.